CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -597,6 +621,11 @@ export default function Report({ reportID }) {
             <div className='border border-slate-400 rounded-xl my-3'>
               <table className='table-auto'>
                 <thead>
+                  <tr>
+                    <th colSpan={7} className="text-2xl">
+                      {currentDisease && currentDiscType ? `${currentDisease} ${currentDiscType}` : 'Report Discrepancies'}
```

Comment:
I like the idea of changing the discrepancy table title when you click on a certain stat in the stats table. However, this title only gets cleared when you click clear filters. If the user changes the filters for a column manually this title isn't updated. Maybe there is a way to where if you change the filters for any column it resets the discrepancy table title back to just "Report Discrepancies".